services/blockstorage/test/harness.go

```diff
@@ -9,24 +9,26 @@ import (
 	"github.com/orbs-network/orbs-network-go/test/harness/services/blockstorage/adapter"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/stretchr/testify/require"
+	"testing"
 )
 
 type driver struct {
 	stateStorage   *services.MockStateStorage
 	storageAdapter adapter.InMemoryBlockPersistence
 	blockStorage   services.BlockStorage
+	t              *testing.T
 }
 
 func (d *driver) expectCommitStateDiff() {
 	csdOut := &services.CommitStateDiffOutput{}
 
 	d.stateStorage.When("CommitStateDiff", mock.Any).Return(csdOut, nil).Times(1)
-
 }
 
 func (d *driver) verifyMocks() {
 	_, err := d.stateStorage.Verify()
-	Expect(err).ToNot(HaveOccurred())
+	require.NoError(d.t, err)
```

Comment:
Let's not store `t` on your driver. 

The convention we use is to pass `t` to `verifyMocks()` as an argument

Reply:
We have multiple calls to t in many driver methods, that's why it's contained in there. We did not have to do it before because we used to have gingko/gomega.